gradoop-data-integration/src/test/resources/data/rdbms/input/husband.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS person(
```

Comment:
Since this is a big importer you should test ALL possible data types. So extend your test case.